stable_baselines3/common/recurrent/policies.py

```diff
@@ -615,3 +621,191 @@ def __init__(
             enable_critic_lstm,
             lstm_kwargs,
         )
+
+
+class RecurrentFeaturesExtractorActorCriticPolicy(BaseRecurrentActorCriticPolicy, Generic[ExtractorInput, RecurrentState]):
+    features_extractor: RecurrentFeaturesExtractor[ExtractorInput, RecurrentState]
+
+    def __init__(
+        self,
+        observation_space: spaces.Space,
+        action_space: spaces.Space,
+        lr_schedule: Schedule,
+        net_arch: Optional[Union[List[int], Dict[str, List[int]]]] = None,
+        activation_fn: Type[nn.Module] = nn.Tanh,
+        ortho_init: bool = True,
+        use_sde: bool = False,
+        log_std_init: float = 0.0,
+        full_std: bool = True,
+        use_expln: bool = False,
+        squash_output: bool = False,
+        features_extractor_class: Type[BaseFeaturesExtractor] = GRUNatureCNNExtractor,
+        features_extractor_kwargs: Optional[Dict[str, Any]] = None,
+        share_features_extractor: bool = True,
+        normalize_images: bool = True,
+        optimizer_class: Type[th.optim.Optimizer] = th.optim.Adam,
+        optimizer_kwargs: Optional[Dict[str, Any]] = None,
+    ):
+        if features_extractor_kwargs is None:
+            features_extractor_kwargs = {}
+        # Automatically deactivate dtype and bounds checks
+        if normalize_images is False and issubclass(features_extractor_class, GRUNatureCNNExtractor):
+            features_extractor_kwargs = features_extractor_kwargs.copy()
+            features_extractor_kwargs.update(dict(normalized_image=True))
```

Comment:
```
features_extractor_kwargs.update(dict(normalized_image=True))
```
seems like a bit of a weird way to accomplish
```
features_extractor_kwargs["normalized_image"]=True
```

But I don't have that strong of an opinion

Reply:
Thanks! f4303434